src/main/java/br/com/deveficiente/bolaoapi/services/championship/Match.java

```diff
@@ -0,0 +1,75 @@
+package br.com.deveficiente.bolaoapi.services.championship;
+
+import br.com.deveficiente.bolaoapi.services.team.Team;
+import lombok.Getter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import javax.validation.constraints.Future;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.time.LocalTime;
+import java.util.Objects;
+
+@ToString(exclude = "championship")
+@Entity
+public class Match {
+
+    @Getter
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Getter
+    @NotNull
+    @ManyToOne
+    private Championship championship;
+
+    @Getter
+    @Positive
+    private Integer round;
+
+    @Getter
+    @NotNull
+    @ManyToOne
+    private Team homeTeam;
+
+    @Getter
+    @NotNull
+    @ManyToOne
+    private Team visitingTeam;
+
+    @Getter
+    @NotNull
+    @Future
+    @Column(columnDefinition = "TIME")
+    private LocalTime startTime;
```

Comment:
Aqui é importante saber o dia do jogo também.

Reply:
Blz! Fui mais pela spec :+1:

Reply:
@asouza por algum motivo o pattern não está rolando (estranho que com LocalDate é super de boas).

Vou enviar LocalTime, e investigar melhor esse caso.

Reply:
Você fala lá no form? Se for, você precisa usar annotation @JsonFormat em cima do atributo.